### DISCUSSION TITLE
Is remote developpement possible ?

### DISCUSSION THREAD

**Discussion by AlexandreBourrieau** [rsta2/circle#290] [Q&A] [answered]
Hello,

I made some tests with Circle and it is awesome thanks !

But burning a sd-card each time I would like to make change is a little borring... So is there a way to use a remote IDE ?
Thanks a lot !

**Comment by rsta2** [comment, marked as answer]
There is a [serial bootloader](https://github.com/rsta2/circle/blob/master/doc/bootloader.txt) and a [network bootloader](https://github.com/rsta2/circle/tree/master/sample/38-bootloader) (via HTTP or TFTP). Also have a look at the [Eclipse support](https://github.com/rsta2/circle/blob/master/doc/eclipse-support.txt) and a [project file for Geany](https://github.com/rsta2/circle/tree/master/tools/template). IDE integration may require some configuration work, but should be doable. Thanks for appreciating Circle!
